### PR TITLE
Account Management + Labels

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch8438]

File: src/app/components/EditableText.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  const hasValue = value !== undefined;
+
+  return (
+    <Box variant="rowAlign" height="100%" {...props}>
+      {editMode ? (
+        <Input
+          placeholder={placeholder}
+          autoFocus={true}
+          value={editValue}
+          onChange={(e) => setEditValue(e.currentTarget.value)}
```
Comment (Mrtenz) [inline src/app/components/EditableText.tsx:56]: Don´t use an arrow function here

File: src/app/components/EditableText.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    onChange(editValue);
+    setEditMode(false);
+  };
+
+  const hasValue = value !== undefined;
+
+  return (
+    <Box variant="rowAlign" height="100%" {...props}>
+      {editMode ? (
+        <Input
```
Comment (Mrtenz) [inline src/app/components/EditableText.tsx:52]: This input is being changed from uncontrolled to controlled when writing something in it. I´m not sure why, but this should be avoided if possible.

**Review by Mrtenz — CHANGES_REQUESTED** [review]
Few things regarding the design:
- I think there should be slightly more padding between the blocky/delete icon and address
- The blockie should be top aligned
- The font size is too big for the address
- The delete overlay should span the entire width

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**